lib/pharos/ssh/remote_file.rb

```diff
@@ -91,12 +91,16 @@ def readlink
         target
       end
 
+      # Returns an array of lines in the remote file
+      # @return [Array<String>]
+      def lines
+        read.lines
+      end
+
       # Yields each line in the remote file
       # @yield [String]
-      def each_line
-        read.split(/[\r\n]/).each do |row|
```

Comment:
(This would cause windows/mac `\r\n` / `\n\r` linefeeds to yield extra blank lines, not an issue here, just noting).